poly-commitment/src/commitment.rs

```diff
@@ -1160,19 +1162,19 @@ pub mod caml {
                 lr: opening_proof
                     .lr
                     .into_iter()
-                    .map(|(g1, g2)| (g1.into(), g2.into()))
+                    .map(|(g1, g2)| (From::from(g1), From::from(g2)))
                     .collect(),
-                delta: opening_proof.delta.into(),
-                z1: opening_proof.z1.into(),
-                z2: opening_proof.z2.into(),
-                sg: opening_proof.sg.into(),
+                delta: From::from(opening_proof.delta),
+                z1: From::from(opening_proof.z1),
+                z2: From::from(opening_proof.z2),
+                sg: From::from(opening_proof.sg),
```

Comment:
why all of these changes?

Reply:
You're right! These aren't necessary. Removing

Reply:
Never mind: an ugly compiler error if these are reverted. Exercise for the reader.